opam

```diff
@@ -18,6 +18,14 @@ depends: [
   "ocamlfind"
   "cstruct" {>= "1.2.0"}
   "zarith"
+  "ounit" {test}
 ]
+
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
```

Comment:
it's a bit unusal to do that (although it would work). See other `build-test` targets in my projects.